Podfile

```diff
@@ -20,11 +20,6 @@ target 'macOS SignalRClient' do
 #  pod 'SwiftWebSocket', '~> 2.7'
 end
 
-target 'SocketsSample' do
```

Comment:
I think it was broken by recent renames and should not be removed. I will update this file shortly.